scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -1,12 +1,23 @@
 function populateCytobands(cytobands){
+  var chromPosPattern = new RegExp("^([0-9]{1,2}|[X|T|MT]{1,2})");
   var chrom = document.forms["filters_form"].elements["chrom"].value;
-  if(chrom===""){
+  var chromPos = document.forms["filters_form"].elements["chrom_pos"].value;
+  var chromosome = "";
+  console.log("Populate cytobands")
+  if(chrom==="" && chromPos===""){
     startElem.value = "";
     endElem.value = "";
     return //only reset cytoband select element
+  // Set the selected chromosome name
+  } else if (chrom!=="" && chromPos!==""){
+    chromosome = chromPos.match(chromPosPattern)[0];
+  } else if (chrom==="" && chromPos!==""){
+    chromosome = chromPos.match(chromPosPattern)[0];
+  } else {
+    chromosome = chrom;
   }
-
-  var chrom_cytobands = cytobands[chrom]["cytobands"]; // chromosome-specific cytobands
+  
+  var chrom_cytobands = cytobands[chromosome]["cytobands"]; // chromosome-specific cytobands
```

Comment:
Updating cytobands according to the values of chrom_pos is nice, but it's not really necessary for the subsequent search, so you can leave it out if you want.